packages/veritone-react-common/src/components/ObjectDetectionEngineOutput/story.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { number } from '@storybook/addon-knobs/react';
+
+import styles from './story.styles.scss';
+import ObjectDetectionOuput from './';
+
+const objectDetectionAssets = [
+  {
+    series: [
+      {
+        startTimeMs: 0,
+        stopTimeMs: 1000,
+        object: {
+          label: 'data',
+          confidence: 0.942457377910614
+        }
+      },
+      {
+        stopTimeMs: 1000,
+        startTimeMs: 0,
+        object: {
+          label: 'very long name 32 char abbreviated',
+          confidence: 0.8848179578781128
+        }
+      },
+      {
+        stopTimeMs: 2000,
+        startTimeMs: 0,
+        object: {
+          label: 'data',
+          confidence: 0.942457377910614
+        }
+      }
+    ]
+  },
+  {
+    series: [
+      {
+        stopTimeMs: 12000,
+        startTimeMs: 9000,
+        object: {
+          label: 'desktop',
+          confidence: 0.8710343837738037
+        }
+      },
+      {
+        stopTimeMs: 11000,
+        startTimeMs: 9000,
+        object: {
+          label: 'data',
+          confidence: 0.8665211200714111
+        }
+      },
+      {
+        stopTimeMs: 10000,
+        startTimeMs: 9000,
+        object: {
+          label: 'data',
+          confidence: 0.899578332901001
+        }
+      },
+      {
+        stopTimeMs: 13000,
+        startTimeMs: 9000,
+        object: {
+          label: 'desktop',
+          confidence: 0.9239837527275085
+        }
+      }
+    ]
+  }
+];
+
+storiesOf('ObjectDetectionEngineOutput', module).add('Base', () => {
```

Comment:
fyi this story is throwing warnings in console when you adjust the knob

Reply:
I'm not seeing this anymore so it must have gotten fixed at some point.